content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -115,14 +79,31 @@ The Istio control plane must be installed with virtual machine integration enabl
     $ istioctl install -f "${WORK_DIR}"/vmintegration.yaml
     {{< /text >}}
 
+## Create Namespace 
+
+1. Create Namespace that will host the virtual machine
+
+    {{< text bash >}}
+    $ kubectl create namespace "${VM_NAMESPACE}"
+    {{< /text >}}
+
+1. Create a serviceaccount for virtual machine
+
+    {{< text bash >}}
+    $ kubectl create serviceaccount "${SERVICE_ACCOUNT}" -n "${VM_NAMESPACE}" 
+    {{< /text >}}
+
 ## Create files to transfer to the virtual machine
 
-1. Make a copy of files to copy to the virtual machine
+1. Create Kubernetes token in the example you can set the  
+    {{< text bash >}}
+    $ tokenexpiretime=3600
+    $ echo '{"kind":"TokenRequest","apiVersion":"authentication.k8s.io/v1","spec":{"audiences":["istio-ca"],"expirationSeconds":'$tokenexpiretime'}}' | kubectl create --raw /api/v1/namespaces/$VM_NAMESPACE/serviceaccounts/$SERVICE_ACCOUNT/token -f - | jq -j '.status.token' > "${WORK_DIR}"/"${CLUSTER_NAME}"/"${VM_NAMESPACE}"/istio-token
+    {{< /text >}}
 
+1. Get the root cert
     {{< text bash >}}
```

Comment:
carriage return between 184 and 185. All text blocks need to be separated by a carrier return in MD.